Client/src/Components/CustomThreshold/index.jsx

```diff
@@ -0,0 +1,92 @@
+import { Box, Stack, Typography } from "@mui/material";
+import Field from "../Inputs/Field";
+import Checkbox from "../Inputs/Checkbox";
+import { useTheme } from "@emotion/react";
+import PropTypes from "prop-types";
+
```

Comment:
This is not a reusable component, you can safely move it to the `createInfrastrucutreMonitor` directory.  the `Components` directory should only hold reusable components

Reply:
sure, will do